src/schema/event.ts

```diff
@@ -22,6 +22,11 @@ export const createEventSchema = z
     path: ["endDate"]
   });
 
+export const addEventRepairerSchema = z.object({
+  userId: z.string().min(1, { message: "User ID is required" }),
+  id: z.string().min(1, { message: "Event ID is required" })
+});
```

Comment:
Not using zod correctly
const name = z.string({
  required_error: "Name is required",
  invalid_type_error: "Name must be a string",
});
z.string().min(5, { message: "Must be 5 or more characters long" });
z.string().max(5, { message: "Must be 5 or fewer characters long" });

ref: https://zod.dev/?id=basic-usage